README.md

```diff
@@ -90,13 +82,13 @@ sudo apt-get install postgresql postgresql-contrib
 Clone this repository to your desired folder:
 
 cd my-folder
-git clone git@github.com:David63011/vet-clinic-database.git
```

Comment:
- [ ] You made reference to the Vet Clnic as the header of the _**README.md**_ file 😄. KIndly change it to the current name of this repository. See the image below for reference:

![image](https://user-images.githubusercontent.com/77106962/217356277-8ab7e9c2-233f-4cff-8799-7009b08cac79.png)

Reply:
fixed